kingpin/actors/aws/cloudformation.py

```diff
@@ -824,11 +842,14 @@ def __init__(self, *args, **kwargs):
         # Check if the supplied CF template is a local file. If it is, read it
         # into memory.
         self._template_body, self._template_url = self._get_template_body(
-            self.option('template'))
+            self.option('template'),
+            self.option('s3_bucket'),
+            self.option('s3_key'),
+            self.option('region')
```

Comment:
This enforces a requirement that the S3 bucket would _have_ to be in the same region as your target stack because you are re-using the `region` parameter to the actor. There should be a separate (optional) key for `template_region` or `template_bucket_region`.